pkgs/applications/science/electronics/librepcb/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, qtbase, qttools, qmake, mesa, openssl, zlib }:
+
+stdenv.mkDerivation rec {
+	name = "librepcb-unstable";
+	version = "2017-12-29";
```

Comment:
usually `-` should not be part of the version. Everything after the last `-` denotes the package version.

Also `version` is not a (formal?) required attribute. Often it is just used as convenient way to share the version string across multiple place within a package.

You should probably rewrite the beginning of your deriviation to look like this:

```
stdenv.mkDerivation rec {
  version = "0.git.20171229";
  name = "librepcb-${version}";
  …
}
```